pkg/k8shandler/dashboards.go

```diff
@@ -0,0 +1,45 @@
+package k8shandler
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path"
+
+	"github.com/openshift/cluster-logging-operator/pkg/utils"
+)
+
+const (
+	clusterLoggingDashboardFile = "dashboards/openshift-logging-dashboard.json"
+)
+
+// CreateOrUpdateDashboards reconciles metrics dashboards component for cluster logging
+func (clusterRequest *ClusterLoggingRequest) CreateOrUpdateDashboards() (err error) {
+	if err = clusterRequest.createOrUpdateClusterLoggingDashboard(); err != nil {
+		return
+	}
+	return nil
+}
+
+// createOrUpdateClusterLoggingDashboard creates/updates the cluster logging dashboard ConfigMap
+func (clusterRequest *ClusterLoggingRequest) createOrUpdateClusterLoggingDashboard() error {
+	b, err := ioutil.ReadFile(path.Join(utils.GetShareDir(), clusterLoggingDashboardFile))
```

Comment:
where is the dashboard file?  there is a util function to grab files from <root>/files.  Ref what we do with fluent/run.sh

Reply:
The util function searches a different dir than this ends up. I believe the util function you're talking about is `GetWorkingDirFileContents`? If so, that's what I tried originally and it searched the wrong directory.

Reply:
https://github.com/openshift/cluster-logging-operator/blob/master/pkg/k8shandler/fluentd.go#L213